examples/stack/link_layer_custom_subrt/example_ll_custom_subrt.py

```diff
@@ -129,9 +128,8 @@ def post_create(conn, q, pair):
                         array_entry = outcomes[fid].get_future_index(pair)
                         q.measure(array_entry)
 
-                    context.epr_sockets[self.PEER].create(
+                    context.epr_sockets[self.PEER].create_keep(
```

Comment:
Again `_keep` is added here and I am not sure why this is done. I see that this is happening at more place, so I think the answer is general, so I won't comment on the other individual cases where this is also done anymore.